dart/lib/src/flx_builder.dart

```diff
@@ -0,0 +1,623 @@
+import 'dart:convert';
```

Comment:
Let's name these files consistently.. `FlexBuffers` is one word, so the lower case would be `flexbuffers` or `flex` not `flx`.

Reply:
OK, will change all the names.